numba/tests/test_analysis.py

```diff
@@ -663,8 +663,7 @@ def _literal_const_sample_generator(self, pyfunc, consts):
         # create new code parts
         co_args = [pyfunc_code.co_argcount]
 
-        if utils.PYVERSION >= (3, 8):
-            co_args.append(pyfunc_code.co_posonlyargcount)
+        co_args.append(pyfunc_code.co_posonlyargcount)
         co_args.append(pyfunc_code.co_kwonlyargcount)
         co_args.extend([pyfunc_code.co_nlocals,
```

Comment:
Now there's no need for a branch this can just be a direct declaration of the entire list?